addons/sale_loyalty/tests/test_program_with_code_operations.py

```diff
@@ -319,6 +320,21 @@ def test_partner_assigned_to_next_order_coupon(self):
             "The partner should be set on the coupon with program type 'next_order_coupons'"
         )
 
+        order = self.empty_order
```

Comment:
A comment that this part is about first unlogged then logged user, could be useful.